modules/ROOT/pages/x12-edi/x12-edi-connector.adoc

```diff
@@ -232,7 +232,7 @@ image::x12-edi/x12-edi-general.jpg[General tab properties]
 The general tab lets you configure settings for reading and writing X12 messages:
 
 * X12 form and validation (standard X12, HIPAA SNIP Type 1, or HIPAA SNIP Type 2).
-* Manually create or edit the list of schema definitions for the message structures to use. They must all use the same form (either X12 or HIPAA). If the list of schema definitions includes a 997 or 999 Functional Acknowledgment schema then that schema will be used to generate Functional Acknowledgments. **Note:** To use the Functional Acknowledgment schema type (997 or 999) it must match the **Generate 999 Functional Acknowledgments** setting on the Parser tab (`false` for 997, `true` for 999).
+* Manually create or edit the list of schema definitions for the message structures to use. They must all use the same form (either X12 or HIPAA). **Note:** To use the Functional Acknowledgment schema type (997 or 999) it must match the **Generate 999 Functional Acknowledgments** setting on the Parser tab (`false` for 997, `true` for 999).
```

Comment:
```suggestion
* Manually create or edit the list of schema definitions for the message structures to use. +
The schema definitions must all use the same form: either X12 or HIPAA. +
To use Functional Acknowledgment schema type 997 or 999, the definition must match the *Generate 999 Functional Acknowledgments* setting on the *Parser tab* (`false` for 997, `true` for 999).
```
Is the Functional Acknowledgement type part of the list of schema definitions, or is this something separate that should perhaps be its own list item?